src/OrchardCore/OrchardCore.DisplayManagement/Descriptors/ShapeAttributeStrategy/ShapeAttributeBindingStrategy.cs

```diff
@@ -47,10 +46,6 @@ public void Discover(ShapeTableBuilder builder)
             {
                 var serviceType = shapeProvider.GetType();
 
-                IFeatureInfo feature = _typeFeatureProvider.GetFeatureForDependency(serviceType);
-                if (builder.ExcludedFeatureIds.Contains(feature.Id))
-                    continue;
```

Comment:
Removed to allow dynamic resolution for attribute shapes

Reply:
I looked in that class this afternoon but it was all chinese and mandarin